rust/common/error/mod.rs

```diff
@@ -97,6 +97,8 @@ error_messages! { TypeQLError
         6: "The query has not been provided with any definables.",
     MatchHasNoBoundingNamedVariable() =
         7: "The match query does not have named variables to bound the nested disjunction/negation pattern(s).",
+    VariableNameConflict(String) =
+        8: "The variable(s) named '{}' cannot be used for both concept variables and value variables.",
```

Comment:
```suggestion
    VariableNameConflict(String) =
        8: "The variable name '{}' cannot be used for both concept variables and value variables.",
```
It's the name that can't be reused, not the variables.

Reply:
Yes, but plural.